reference/JsParseModuleSource.md

```diff
@@ -0,0 +1,28 @@
+Parse the source for an ES module
+### Syntax 
+```
+CHAKRA_API
+JsParseModuleSource(
+    _In_ JsModuleRecord requestModule,
+    _In_ JsSourceContext sourceContext,
+    _In_ BYTE* script,
+    _In_ unsigned int scriptLength,
+    _In_ JsParseModuleSourceFlags sourceFlag,
+    _Outptr_result_maybenull_ JsValueRef* exceptionValueRef);
+```
+### Parameters 
+* __requestModule__: The ModuleRecord being parsed.
+* __sourceContext__: A cookie identifying the script that can be used by debuggable script contexts.
+* __script__: The source script to be parsed, but not executed in this code.
+* __scriptLength__: The source length of sourceText. The input might contain embedded null.
```

Comment:
Might be worth specifying that this is the length in bytes, not chars, since this method accepts both utf8 and utf16 inputs.